app.py

```diff
@@ -311,60 +389,56 @@ def main():
+            main_chart_data, collateral_token_price = process_liquidity(
+                main_chart_data, collateral_token, debt_token
+            )
     else:
-        main_chart_data, collateral_token_price = process_liquidity(main_chart_data, collateral_token, debt_token)
+        main_chart_data, collateral_token_price = process_liquidity(
+            main_chart_data, collateral_token, debt_token
+        )
 
     # TODO: Add Ekubo end
     figure = src.main_chart.get_main_chart_figure(
         data=main_chart_data,
         collateral_token=collateral_token,
-        debt_token=src.settings.STABLECOIN_BUNDLE_NAME if current_pair == stable_coin_pair else debt_token,
+        debt_token=src.settings.STABLECOIN_BUNDLE_NAME
+        if current_pair == stable_coin_pair
+        else debt_token,
         collateral_token_price=collateral_token_price,
     )
     streamlit.plotly_chart(figure_or_data=figure, use_container_width=True)
 
-    main_chart_data['debt_to_supply_ratio'] = (
-        main_chart_data['liquidable_debt_at_interval'] / main_chart_data['debt_token_supply']
+    main_chart_data["debt_to_supply_ratio"] = (
+        main_chart_data["liquidable_debt_at_interval"]
+        / main_chart_data["debt_token_supply"]
     )
     example_rows = main_chart_data[
-        (main_chart_data['debt_to_supply_ratio'] > 0.75)
-        & (main_chart_data['collateral_token_price'] <= collateral_token_price)
+        (main_chart_data["debt_to_supply_ratio"] > 0.75)
+        & (main_chart_data["collateral_token_price"] <= collateral_token_price)
     ]
 
     if not example_rows.empty:
-        example_row = example_rows.sort_values('collateral_token_price').iloc[-1]
+        example_row = example_rows.sort_values("collateral_token_price").iloc[-1]
 
         def _get_risk_level(debt_to_supply_ratio: float) -> str:
             if debt_to_supply_ratio < 0.2:
-                return 'low'
+                return "low"
             elif debt_to_supply_ratio < 0.4:
-                return 'medium'
+                return "medium"
             elif debt_to_supply_ratio < 0.6:
-                'high'
-            return 'very high'
+                "high"
+            return "very high"
 
         streamlit.subheader(
             f":warning: At price of {round(example_row['collateral_token_price'], 2)}, the risk of acquiring bad debt for "
             f"lending protocols is {_get_risk_level(example_row['debt_to_supply_ratio'])}."
-        )    
+        )
         streamlit.write(
             f"The ratio of liquidated debt to available supply is {round(example_row['debt_to_supply_ratio'] * 100)}%.Debt"
             f" worth of {int(example_row['liquidable_debt_at_interval']):,} USD will be liquidated while the AMM swaps "
             f"capacity will be {int(example_row['debt_token_supply']):,} USD."
         )
 
-    streamlit.header("Liquidable debt")
```

Comment:
Seems like you've accidentally deleted valid code!

Reply:
"@lukaspetrasek, I am currently facing some challenges with resolving the conflicts in this PR. Would it be acceptable to close this PR, re-implement my changes, and submit a new one?"